tmp/build/Dockerfile

```diff
@@ -3,6 +3,6 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 ARG build_type=maistra
 
 ADD tmp/_output/bin/istio-operator /usr/local/bin/istio-operator
-ADD tmp/_output/templates/ /usr/share/istio-operator/default-templates
+ADD tmp/_output/templates/ /usr/local/share/istio-operator/default-templates
 ADD manifests-${build_type} /manifests
-COPY tmp/_output/helm/istio-releases/istio-1.1.0 /etc/istio-operator/1.1.0/helm
+COPY tmp/_output/helm/istio-releases/istio-1.1.0 /usr/local/share/istio-operator/helm
```

Comment:
I think we need to keep the version in the directory name, as we'll eventually have multiple versions in there.

Reply:
@luksa  Do you think we should do this now or can we leave that for the future, when we proper implement it?

Reply:
Since we had the version in so far, I'd say it's best to keep it.